ocp_resources/cluster_service_version.py

```diff
@@ -6,3 +9,15 @@ class ClusterServiceVersion(NamespacedResource):
 
     class Status(NamespacedResource.Status):
         INSTALLING = "Installing"
+
+    def get_dicts_from_examples(self) -> List[dict[str, Any]]:
```

Comment:
I case that we don;t have "olm-example" you return List and not `List[dict[str, Any]]` as the function state